.github/integration_test.sh

```diff
@@ -87,7 +93,28 @@ if ! grep --silent "$SUCCESS_REGEX" example-lein-output; then
   exit 1
 fi
 
-# 5.- Dogfood the `nvd-clojure` project
+# 5.- Exercise Clojure CLI Tools integration
+
+cd "$PROJECT_DIR/example" || exit 1
+
+example_classpath="$(clojure -Spath)"
+
+# cd to $HOME, to demonstrate that the Tool does not depend on a deps.edn file:
+cd || exit 1
+
+if clojure -Tnvd nvd.task/check :classpath '"'$example_classpath'"' > example-lein-output; then
```

Comment:
I had to add two tweaks over what was hinted over the readme:

* had to qualify `nvd.task/check`, the CLI wouldn't be happy otherwise
* changed `io.github` Maven prefix for `nvd-clojure`, which matches our project.clj structure

Reply:
Re: qualification -- that indicates you don't have the correct `:tools/usage` in `deps.edn` to specify the default namespace. Ah, most likely because you're installing from the Clojars version and it doesn't have `deps.edn` in the JAR?

Normally, CLI tool installation is done from GitHub, not Clojars/Maven, and you use `io.github.rm-hull/nvd-clojure` as the lib name (group/artifact => GitHub URL) and just specify a tag `{:git/tag "v1.2.3"}`